From 49abb326542984cecc5e61b3b66822b52cdc34e7 Mon Sep 17 00:00:00 2001
From: openEuler Buildteam <buildteam@openeuler.org>
Date: Mon, 30 Dec 2019 14:49:06 +0800
Subject: [PATCH] change lib to lib64

---
 Configure | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/Configure b/Configure
index 3be9f05..1c53af7 100755
--- a/Configure
+++ b/Configure
@@ -7269,8 +7269,8 @@ esac'
 : Reproduce behavior of 5.005 and earlier, maybe drop that in 5.7.
 case "$installstyle" in
 '')	case "$prefix" in
-		*perl*) dflt='lib';;
-		*) dflt='lib/perl5' ;;
+		*perl*) dflt='lib64';;
+		*) dflt='lib64/perl5' ;;
 	esac
 	;;
 *)	dflt="$installstyle" ;;
@@ -7336,8 +7336,8 @@ esac
 : /opt/perl/lib/perl5... would be redundant.
 : The default "style" setting is made in installstyle.U
 case "$installstyle" in
-*lib/perl5*) set dflt privlib lib/$package/$version ;;
-*)	 set dflt privlib lib/$version ;;
+*lib64/perl5*) set dflt privlib lib64/$package/$version ;;
+*)	 set dflt privlib lib64/$version ;;
 esac
 eval $prefixit
 $cat <<EOM
@@ -7584,8 +7584,8 @@ siteprefixexp="$ansexp"
 prog=`echo $package | $sed 's/-*[0-9.]*$//'`
 case "$sitelib" in
 '') case "$installstyle" in
-	*lib/perl5*) dflt=$siteprefix/lib/$package/site_$prog/$version ;;
-	*)	 dflt=$siteprefix/lib/site_$prog/$version ;;
+	*lib64/perl5*) dflt=$siteprefix/lib64/$package/site_$prog/$version ;;
+	*)	 dflt=$siteprefix/lib64/site_$prog/$version ;;
 	esac
 	;;
 *)	dflt="$sitelib"
@@ -8001,8 +8001,8 @@ case "$vendorprefix" in
 	'')
 		prog=`echo $package | $sed 's/-*[0-9.]*$//'`
 		case "$installstyle" in
-		*lib/perl5*) dflt=$vendorprefix/lib/$package/vendor_$prog/$version ;;
-		*)	     dflt=$vendorprefix/lib/vendor_$prog/$version ;;
+		*lib64/perl5*) dflt=$vendorprefix/lib64/$package/vendor_$prog/$version ;;
+		*)	     dflt=$vendorprefix/lib64/vendor_$prog/$version ;;
 		esac
 		;;
 	*)	dflt="$vendorlib"
-- 
1.8.3.1

